packages/block-editor/src/hooks/use-bindings-attributes.js

```diff
@@ -63,15 +98,19 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 		);
 		const bindings = props.attributes.metadata?.bindings;
 		const { name, clientId, context } = props;
+		const bindingsWithDefaults = useMemo(
+			() => replacePatternOverrideDefaultBindings( name, bindings ),
+			[ bindings, name ]
+		);
```

Comment:
Could we still call this `bindings` and pass directly `props.attributes.metadata?.bindings`? I feel it makes the rest of code easier to read and in the end it is just the bindings object.

Reply:
Yep, I thought about that too. It also reduces the diff. I've made that change 👍